### PR TITLE
fix travis golang issue

### PR DESCRIPTION
**Pull request by mikebrow**
Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 language: go
 
+go:
+  - "1.10"
```
Comment (Random-Liu) [inline .travis.yml]: We don't need this after removing `-` I guess.

**Comment by Random-Liu** [comment]
You also need to remove the `-` under the second stage

File: .travis.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ before_script:
 jobs:
   include:
     - stage: Build
-    - script:
+      script:
```
Comment (Random-Liu) [inline .travis.yml]:
```
     - stage: Test
     - script:
```
->
```
     - stage: Test
       script:
```

Comment (mikebrow) [reply .travis.yml]: yeah tried it that way once and didn't get the right result..

Comment (mikebrow) [reply .travis.yml]: probably because I had not yet removed the -go.. looks good now

**Comment by mikebrow** [comment]
Looks good now...

**Pull request merged**